tests/console/yast2_i.pm

```diff
@@ -54,6 +54,17 @@ sub run {
         }
         wait_screen_change { send_key 'alt-p' };
     }
+
+    # in new yast2 sw_install we need to change filter to Search
+    if (is_tumbleweed) {
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

Reply:
I believe we'll get this change to SLE 15 SP1 and leap 15.1, what's about using `yast2-preselected-driver` tag for the detection?

Reply:
It is not only in the preselect package, but we need to navigate to filter and select search option as well. Search has been the default opt until the newest updates.

Reply:
ok, could you still negate the check and do e.g. `if (!is_leap('<=15.1') && !is_sle('<=15-SP1'))`

Reply:
I think it is easier and more clear if we include only the bare minimum in the condition. I guess that we can expect this change will hit the newest leap and sle15sp1 anyway.